### ISSUE TITLE
Module partially works with newer BVM model (hx310), but not quite...

### ISSUE THREAD

**Issue by Cledwyn** [bitfocus/companion-module-sony-broadcastmonitor#3]
Function keys in particular do not appear to correctly map/behave as expected.
Could this module be updated to support this model?

https://pro.sony/en_GR/support-resources/bvm-hx310/manual

**Comment by daniep01** [comment]
I'm sure it could be resolved if I could get a copy of the remote control protocol for this model. The manual posted above does not have the required information.

**Comment by Cledwyn** [comment]
Yeah, annoying - It's a little strange - As colleagues are telling me that the older control units (BKM-16R) work fine with this BVM model, including the function keys

**Comment by Cledwyn** [comment]
Based on some recent convos, I think that the old Sony Monitor control guide may still be the relevant reference.
I've not had time to try testing any commands directly, myself.
I guess that the features may have only been partially implemented so far in this module, or perhaps tested on a very specific model?

Original module request: https://github.com/bitfocus/companion-module-requests/issues/468
Network Control Document: https://github.com/bitfocus/companion-module-requests/files/6274240/Sony.Professional.Video.Monitor.Network.Control.Manual_EN_V100.pdf

**Comment by daniep01** [comment]
I would like to solve this if possible but first the module needs to be re-written for Companion3. The original module was sponsored so some extra funding will need to be provided in order for me to work on it.
